README.md

```diff
@@ -78,6 +80,7 @@ Ruby Specific Mappings
 \rf - run the spec under the cursor
 \rl - run the last spec
 \rs - syntax check the current file
+Ctrl+L (in insert mode) - insert a " => " at the current cursor position
```

Comment:
Lowercase `l`, right?

Reply:
Yeah, though that felt more confusing to me... I think people largely understand it's lowercase cause there's no `Shift` there?

Reply:
I assume capital letters in shortcuts imply a shift, but I don't know if that's universal.